### DISCUSSION TITLE
GitHub Copilot unable to process application

### DISCUSSION THREAD

**Discussion by nillawafers11** [community/community#51874] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello, I'm a student at a university. After updating my Account information (Name, Bio), the application page tells me:

-Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
-You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I attend my university in person, albeit I do not live on campus. However, I suspect when I was asked for Location services, it recognizes I am in Chicago, but that is merely because my ISP (Starlink) puts me there. What can I do to fix this and continue applying for Copilot?

Thanks!

**Comment by pierizvi** [comment, marked as answer]
You can use different ISP to fix your IP address specific to campus location. I would suggest apply it in your UNIVERSITY using campus Wifi to get specifics accurate.

**Reply by nillawafers11** [reply]
Thank you, I'll be on campus later today. Cheers!

**Reply by pierizvi** [reply]
If you face any difficulty don't hesitate to ask 🤗
